Tests/FluentAssertions.Specs/Primitives/DateOnlyAssertionSpecs.cs

```diff
@@ -0,0 +1,863 @@
+using System;
+using Xunit;
+using Xunit.Sdk;
+
+#if NET6_0_OR_GREATER
+
+namespace FluentAssertions.Specs.Primitives
+{
+    public class DateOnlyAssertionSpecs
+    {
+        #region (Not) Be
```

Comment:
🔧 We should be using the nested class construct as we've been doing recently. See also https://github.com/fluentassertions/fluentassertions/issues/1340